sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+    "    output_file_name=output_file,\n",
+    "    retry_settings=BatchRetrySettings(max_retries=3, timeout=30),\n",
+    "    logging_level=\"info\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, start a model deployment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.begin_create_or_update(batch_deployment).wait()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We need to create the Input, representing URI folder, because the batch endpoint is intended to process multiple files at a time. In this example we will use only one test file, we have uploaded to the blob storage before. This file must be available through the url link.\n",
+    "\n",
+    "Create an inference job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = ml_client.batch_endpoints.invoke(\n",
+    "    endpoint_name=batch_endpoint_name,\n",
+    "    input=my_test_data_input,\n",
+    "    deployment_name=\"non-mlflow-deployment\",  # name is required as default deployment is not set\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will stream the job output to monitor the execution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = job.name\n",
+    "batch_job = ml_client.jobs.get(name=job_name)\n",
+    "print(batch_job.status)\n",
+    "# stream the job logs\n",
+    "ml_client.jobs.stream(name=job_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.2 Download the prediction result for metrics calculation\n",
+    "The output of prediction is saved in JSON format. You can use it to calculate test set metrics and plot predictions and actuals over time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(job_name, download_path=\".\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fcst_df = pd.read_json(output_file, orient=\"table\")\n",
+    "fcst_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calculate the metrics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from helpers.metrics_helper import calculate_metrics\n",
+    "\n",
+    "calculate_metrics(fcst_df[target_column_name], fcst_df[\"predicted\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.3 Forecast versus actuals plot"
```

Comment:
Please change the section headers including section numbers to match the energy demand notebook.